CMakeLists.txt

```diff
@@ -60,8 +60,14 @@ if (NOT DEFINED CPACK_PACKAGE_CONTACT)
 endif()
 
 if (NOT DEFINED CPACK_PACKAGE_DESCRIPTION_SUMMARY)
-set(CPACK_PACKAGE_DESCRIPTION_SUMMARY
-	"GENOA™ LCD display Interface library")
+    # Conditionally set the description summary based on the selected version
+    if (USE_OLED_VERSION)
+        set(CPACK_PACKAGE_DESCRIPTION_SUMMARY
+            "FUTURE™ LCD display Interface library")
```

Comment:
instead of "FUTURE" should we say "SP7"?

Reply:
Yes can change this